### PR TITLE
Working branch

### PR DESCRIPTION
**Pull request by ignaciotb**
Look at me! I'm requesting a pull!
Good luck checking it out, it's huge

### CODE REVIEW

File: ekf_general/DataSets/.svn/all-wcprops [hunk 1]
```diff
@@ -0,0 +1,41 @@
+K 25
```
Comment (nilsbore) [inline ekf_general/DataSets/.svn/all-wcprops:1]: @ignaciotb Did you mean to check in all these files in the .svn directory?

File: ekf_lolo_auv/src/landmark_ml.cpp [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    landmark_pos_(2) = landmark_pos(3);
+}
+
+void LandmarkML::computeH(const boost::numeric::ublas::vector<double> &mu_hat,
+                          const tf::Vector3 lm_odom){
+
+    using namespace std;
+    H_ = boost::numeric::ublas::identity_matrix<double>(2,6);
+
+    H_(0,0) = cos(mu_hat(3))*sin(mu_hat(5)) - cos(mu_hat(5))*sin(mu_hat(4))*sin(mu_hat(3));
```
Comment (nilsbore) [inline ekf_lolo_auv/src/landmark_ml.cpp]: Holy #"?¤=?#¤, I'm gonna trust you on this one...:)

**Review by nilsbore — COMMENTED** [review]
Just some preliminary stuff, I'll look some more when I have time.

File: ekf_general/DataSets/.svn/all-wcprops (hunk 1 above)
Comment (ignaciotb) [reply ekf_general/DataSets/.svn/all-wcprops:1]: nop! The folder was hidden

File: ekf_lolo_auv/src/landmark_ml.cpp (hunk 2 above)
Comment (ignaciotb) [reply ekf_lolo_auv/src/landmark_ml.cpp]: It's something matlab pucked and I copy pasted. But it works!

**Pull request merged**